pycrdt_websocket/ystore.py

```diff
@@ -361,7 +359,7 @@ async def start(
 
     async def stop(self) -> None:
         """Stop the store."""
-        if self.db_initialized.is_set():
+        if hasattr(self, "db_initialized") and self.db_initialized.is_set():
```

Comment:
I don't like that, `db_initialized` should be set to `None` in `__init__` and you should check for `None` here instead of looking up for an attribute on the class instance:
```py
if self.db_initialized is not None and self.db_initialized.is_set():
```